x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.test.ts

```diff
@@ -147,6 +147,21 @@ describe('rules_notification_alert_type', () => {
         total: { value: 10 },
       },
     });
+    // @ts-expect-error missing HTTP response stuff. This is fine we don't care about meta and path etc..
+    alertServices.scopedClusterClient.fieldCaps.mockResolvedValue({
```

Comment:
Appreciate the reasoning from the ts-expect-error! Nice.

fwiw, you can do this and get a bit more type safety. It looked like it worked for me:

```ts
    const value: Partial<ApiResponse> = {
      statusCode: 200,
      body: {
        fields: {
          '@timestamp': {
            date: {
              indices: ['index1', 'index2', 'index3', 'index4'],
              searchable: true,
              aggregatable: false,
            },
          },
        },
      },
    };
    alertServices.scopedClusterClient.fieldCaps.mockResolvedValue(value as ApiResponse);
```

That gives you a bit better safety and gives us a bit better chance of seeing problems if the shape of the data changes down the road.

Optional here but I wanted to point it out.